frontend/src/components/userProfile/UserProfileForm.tsx

```diff
@@ -35,106 +37,191 @@ const UserProfileForm = ({
-        <Textarea
-          height="150px"
-          type="experience"
-          onChange={(e) => setLanguageExperience(e.target.value)}
-        />
-      </FormControl>
-      {isSignup && (
-        <>
+        {isSignup && (
+          <Text marginBottom="20px">
+            For the purposes of this platform, please enter your details
+          </Text>
+        )}
+        <FormControl>
           <Heading marginTop="24px" size="sm">
-            Select language*
+            Full name*
           </Heading>
-          <Text marginBottom="12px">
-            Select the language that you would like to start with. Additional
-            languages can be added after signing up.
-          </Text>
-          <Box width="60%" marginBottom="200px">
-            <Select
-              placeholder="Select language"
-              options={languageOptions}
-              getOptionLabel={(option: any) => `
-                ${convertLanguageTitleCase(option.value || "")}
-              `}
-              components={{ DropdownIndicator }}
+          <Input
+            type="name"
+            value={fullName}
+            onChange={(e) => setFullName(e.target.value)}
+          />
+          <Heading marginTop="24px" size="sm">
+            Email*
+          </Heading>
+          <Input
+            type="email"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+          />
+          <Heading marginTop="24px" size="sm">
+            Educational qualifications*
+          </Heading>
+          <Textarea
+            height="150px"
+            type="qualifications"
+            onChange={(e) => setEducationalQualification(e.target.value)}
+          />
+          <Heading marginTop="24px" size="sm">
+            Language experience*
+          </Heading>
+          <Textarea
+            height="150px"
+            type="experience"
+            onChange={(e) => setLanguageExperience(e.target.value)}
+          />
+        </FormControl>
+        {isSignup && (
+          <>
+            <Heading marginTop="24px" size="sm">
+              Select language*
+            </Heading>
+            <Text marginBottom="12px">
+              Select the language that you would like to start with. Additional
+              languages can be added after signing up.
+            </Text>
+            <Box width="60%" marginBottom="200px">
+              <Select
+                placeholder="Select language"
+                options={languageOptions}
+                getOptionLabel={(option: any) => `
+                  ${convertLanguageTitleCase(option.value || "")}
+                `}
+                components={{ DropdownIndicator }}
+              />
+            </Box>
+          </>
+        )}
+      </Flex>
+      <Flex direction="column" flex={1} marginLeft="60px" marginTop="75px">
+        <Heading size="sm">Upload your CV (optional)</Heading>
+        <Flex
+          align="center"
+          background="#e8f0f6"
```

Comment:
whats this colour supposed to be? is it one of our system colours at a different opacity?

Reply:
This seems like it was there before. I'll change it to `blue.50`.

Reply:
thnx!